tests/expecter.go

```diff
@@ -228,7 +229,14 @@ func ExpectBatchWithValidatedSend(expecter expect.Expecter, batch []expect.Batch
 
 			// Remove the \n since it is translated by the console to \r\n.
 			previousSend = strings.TrimSuffix(previousSend, "\n")
-			bExp.R = fmt.Sprintf("%s%s%s", previousSend, "((?s).*)", bExp.R)
+			// Somtimes after logging in, the previous command will be removed from the buffer,
+			// for that we allow it also to match without the previous command, but only if it is at
+			// the beggining of the line.
+			// Plus in some consoles the prompt is read with a preceded white space, and sometimes it doesn't, so we
+			// allow that as well.
+			bExp.R = fmt.Sprintf("(%s%s%s"+`|^\s?`+"%s*%s)", previousSend, "((?s).*)", bExp.R, CRLF, bExp.R)
+			//avoid a literal \r in the middle of a command.
```

Comment:
can you reword this comment? it may be stale.

Reply:
Done.